src/main/scala/cognite/spark/v1/DefaultSource.scala

```diff
@@ -40,18 +40,35 @@ final case class RelationConfig(
   def sparkPartitions: Int = Math.max(1, partitions / parallelismPerPartition)
 }
 
-object OnConflict extends Enumeration {
-  type Mode = Value
-  val Abort, Update, Upsert, Delete = Value
-
-  def withNameOpt(s: String): Option[Value] = values.find(_.toString.toLowerCase == s.toLowerCase())
+sealed trait OnConflictOption extends Serializable
```

Comment:
Scala 2.13 doesn't like exhaustiveness checking on `Enumeration`, and they're a bit awkward anyways, so using a sealed trait with objects instead.

Reply:
I was about to ask =))